src/data/trainer-title.csv

```diff
@@ -205,6 +207,7 @@
 "キミの夢へと走り出せ！","Chase Your Dreams"
 "Brand-new Friend","Brand-new Friend"
 "花咲く乙女のJunePride","JunePride of the Blooming Maidens"
+"幻想世界ウマネスト","Fantasy World Umanest"
```

Comment:
"Uma Nest" would be better here I think, since this whole event is a reference to Dragon Quest.

Reply:
Resolved via 6f87e9f; please update your review status